.github/workflows/ci.yml

```diff
@@ -19,22 +20,30 @@ jobs:
 
   lint:
     name: Lint
-    runs-on: windows-2019
+    runs-on: windows-2022
     steps:
       - uses: actions/checkout@v4
         with:
           submodules: true
       - run: rustup toolchain install stable --profile minimal --component clippy --no-self-update
+      - name: Install LLVM and Clang
+        uses: KyleMayes/install-llvm-action@v2
+        with:
+          version: ${{ env.LLVM_VERSION }}
       - uses: Swatinem/rust-cache@v2
       - run: cargo clippy --all-features -- -D warnings
 
   test:
     name: Test
-    runs-on: windows-2019
+    runs-on: windows-2022
```

Comment:
I would suggest to go all the way to `windows-2025` if you are going to update it.